render_test.go

```diff
@@ -22,7 +23,7 @@ func TestRenderPageMethod(t *testing.T) {
 			renderer.BuildPage(w, mockPage, "test-homepage")
 
 			Convey("Then the render client's HTML method should be called, and the page model should have assets path and site domain injected", func() {
-				So(mockClient.ValidBuildHTMLMethodCalls, ShouldEqual, 1)
+				So(len(mockClient.ValidBuildHTMLMethodCalls), ShouldEqual, 1)
```

Comment:
Could use `ShouldHaveLength`
```suggestion
				So(mockClient.ValidBuildHTMLMethodCalls, ShouldHaveLength, 1)
```

Reply:
👍